cargo-dist/src/host.rs

```diff
@@ -350,11 +353,15 @@ pub(crate) fn select_hosting(
     let repo = raw_repository_url
         .github_repo()
         .map_err(|e| DistError::CantEnableGithubUrlNotGithub { inner: e })?;
+    let domain = repo.domain();
     let repo_url = repo.web_url();
```

Comment:
do we keep this if we are constructing the others?

Reply:
Looks like it's fully unused now, yeah. I'll cut it.